app/models/game.rb

```diff
@@ -1,9 +1,37 @@
 class Game < ApplicationRecord
-  belongs_to :white_player, class_name: 'User'
-  belongs_to :black_player, class_name: 'User', required: false
+
+  after_create :populate_game
+  # belongs_to :white_player, class_name: "Player"
+  # belongs_to :black_player, class_name: "Player"
   has_many :pieces
 
-  def self.needs_second_player
-    Game.where(status: 'Open')
+  def populate_game
```

Comment:
The rubocop checks that attempt to limit method complexity are failing this method since it's so long.  There pretty much isn't a way to get around that here though, so can you add the following comments before and after this method, to disable those checks just for this?

```ruby
# rubocop:disable MethodLength
# rubocop:disable Metrics/AbcSize
def populate_game
  ...
end
# rubocop:enable MethodLength
# rubocop:enable Metrics/AbcSize